.github/workflows/debug-artifacts-failure.yml

```
@@ -35,6 +35,9 @@ jobs:
         uses: ./.github/prepare-test
         with:
           version: latest
+      - uses: actions/setup-go@v3
+        with:
+        go-version: ^1.13.1
```

I think this is why these checks aren't running.
```suggestion
          go-version: ^1.13.1
```